libs/core/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.html

```diff
@@ -13,10 +13,15 @@
                 <ng-content select="fd-facet"></ng-content>
             }
             <div class="fd-dynamic-page__title-subtitle-container">
-                <h2 class="fd-dynamic-page__title-container">
+                <div class="fd-dynamic-page__title-container">
                     <span
+                        tabindex="0"
+                        role="heading"
+                        aria-level="2"
```

Comment:
I think this probably needs to be either aria-level 1 (assuming the page will be fullscreen), or, configurable via input by the dev